features/nested_test_2.feature

```diff
@@ -0,0 +1,5 @@
+Feature: General Feature Test - Nested Test 2
+
+  Scenario: Test if general steps are loaded from the 'nested_steps' subdirectory
+    Given a general step is executed in sub sub folder 2
```

Comment:
Thanks for the feedback! 
My main goal was just to verify that the steps are passing correctly and ensure there are no conflicts between folders, rather than strictly following the Given-When-Then structure.
That said, I can update it to "When" if it makes more sense from a BDD perspective. 
Appreciate your help!